entries/w/woolworths.com.au.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Woolworths": {
+    "domain": "woolworths.com.au",
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "notes": "A phone number and email are required for account setup. 2FA is turned on by default.",
```

Comment:
Like with a similar PR you submitted, the information isn't really relevant as it isn't a restriction nor an exception.
Good on you for being really thorough, though! :+1: 
```suggestion
```